test/functional/tests/conftest.py

```diff
@@ -113,15 +113,16 @@ def pytest_runtest_teardown():
 
 
 def pytest_configure(config):
+    add_marks(config)
     TestRun.configure(config)
 
 
 def pytest_addoption(parser):
     parser.addoption("--dut-config", action="store", default="None")
     parser.addoption("--log-path", action="store",
                      default=f"{os.path.join(os.path.dirname(__file__), '../results')}")
-    parser.addoption("--force-reinstall", action="store", default="False")
-    # TODO: investigate whether it is possible to pass the last param as bool
+    parser.addoption("--force-reinstall", action="store", default=False)
```

Comment:
Did you verify if it works? I remember that we wanted to use bool type here but for some reason it didn't work and we decided to use string here.

Reply:
I found this:
https://docs.pytest.org/en/latest/reference.html#parser
https://docs.python.org/3/library/argparse.html#action